packages/rocketchat-api/server/v1/users.js

```diff
@@ -270,3 +270,66 @@ RocketChat.API.v1.addRoute('users.createToken', { authRequired: true }, {
 		return data ? RocketChat.API.v1.success({data}) : RocketChat.API.v1.unauthorized();
 	}
 });
+
+RocketChat.API.v1.addRoute('users.getPreferences', { authRequired: true }, {
+	get() {
+		const user = this.isUserFromParams() ? RocketChat.models.Users.findOneById(this.userId) : this.getUserFromParams();
```

Comment:
With that any use will be able to get information about other users?

Reply:
well, not really. this is only to validate that the user required on the params actually exists. but there's validation in place. if you check most of the other methods on this file, they all do it this way, I just followed the pattern, which is pretty safe.